apluslms_roman/cache_file.py

```diff
@@ -0,0 +1,28 @@
+from os import remove
+from os.path import join
+
+from apluslms_yamlidator.document import Document
+from apluslms_yamlidator.validator import ValidationError
+from apluslms_yamlidator.utils.version import Version
+
+from . import CACHE_DIR
+
+class CacheFile(Document):
+    name = 'roman_cache'
+    schema = name
+    version = Version(1, 0)
```

Comment:
I don't think these need to be here if this is only a class that implements loader for who ever uses it. For comparison, there are classes in settings.py.